cpu/esp32/include/periph_cpu_esp32h2.h

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (C) 2025 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#pragma once
+
+/**
+ * @ingroup     cpu_esp32
+ * @{
+ *
+ * @file
+ * @brief       ESP32-H2 specific peripheral configuration
+ *
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#include "sdkconfig.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Mapping configured ESP32-H2 default clock to CLOCK_CORECLOCK define */
+#define CLOCK_CORECLOCK     (1000000UL * CONFIG_ESP_DEFAULT_CPU_FREQ_MHZ)
```

Comment:
```suggestion
#define CLOCK_CORECLOCK     MHZ(CONFIG_ESP_DEFAULT_CPU_FREQ_MHZ)
```

We have a macro for that :)

Reply:
I know. I'm not sure but if I remember correcly, we didn't use `MHZ` here because it required to include of `macros/units.h` in some cases which in turn led to a redefinition conflict with the `MHZ` macro already defined in a header file of the SDK. Lets try to see what CI says.